quic/s2n-quic-sim/Cargo.toml

```diff
@@ -23,16 +23,9 @@ s2n-quic = { path = "../s2n-quic", features = ["unstable-provider-io-testing", "
 s2n-quic-core = { path = "../s2n-quic-core", features = ["testing"] }
 serde = { version = "1", features = ["derive"] }
 serde_json = "1"
-# >0.6.1 breaks compatibility with 1.63
-# See https://github.com/aws/s2n-quic/issues/1735
-serde_spanned = "=0.6.1"
+serde_spanned = "0.6"
```

Comment:
Most of these pinned dependencies aren't actually used directly in the crate. I would start by removing them all and see what breaks.